lib/src/rules/pub/package_descriptions.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:linter/src/analyzer.dart';
+import 'package:linter/src/utils.dart';
+
+const _desc = r'Use brief package descriptions.';
+
+const _details = r'''
+
+**DO** keep package descriptions between 60 and 180 characters long.
```

Comment:
I think it would be good to include the fact that this rule exists because of a policy defined by `pana` (though it should be described in terms of `pub.dev`, as most users won't know what `pana` is).

Have we coordinated with the maintainers of pana so that they know that this lint needs to be updated if they change their policy?

I agree that the 60 character limit is odd; perhaps you could create an issue in `pana` asking the question there.